astroid/modutils.py

```diff
@@ -66,22 +66,7 @@
     PY_COMPILED_EXTS = ("so",)
 
 
-try:
-    # The explicit sys.prefix is to work around a patch in virtualenv that
-    # replaces the 'real' sys.prefix (i.e. the location of the binary)
-    # with the prefix from which the virtualenv was created. This throws
-    # off the detection logic for standard library modules, thus the
-    # workaround.
-    STD_LIB_DIRS = {
-        get_python_lib(standard_lib=True, prefix=sys.prefix),
-        # Take care of installations where exec_prefix != prefix.
-        get_python_lib(standard_lib=True, prefix=sys.exec_prefix),
-        get_python_lib(standard_lib=True),
-    }
-# get_python_lib(standard_lib=1) is not available on pypy, set STD_LIB_DIR to
-# non-valid path, see https://bugs.pypy.org/issue1164
-except DistutilsPlatformError:
-    STD_LIB_DIRS = set()
+STD_LIB_DIRS = {sysconfig.get_path("stdlib")}
```

Comment:
Should we include `sysconfig.get_path("platstdlib")` too?
That would match `get_python_lib(standard_lib=True, prefix=sys.prefix)`.

```py
{
 'platstdlib': '/home/runner/work/astroid/astroid/venv/lib/python3.10',
 'stdlib': '/opt/hostedtoolcache/Python/3.10.2/x64/lib/python3.10'
}
```

Usually `stdlib` should be enough. I just don't know if `virtualenv` or others do something unexpected with the lib files.

Reply:
I'm not sure? It was added in:
https://github.com/PyCQA/astroid/commit/b8c48252e042659e009cb95b87c2db9bb225d4fb
But there was not test or related issue that prompted the change?

Reply:
It would had to be a virtualenv issue from the looks of it. Tbh I don't know if it has been fixed in the meantime. Maybe we should include in for now, just to be safe. The additional path doesn't really hurt.

If we really want to, we could revisit this in the future and just remove it then to see if it causes any new regressions. I just wouldn't suggest to do it now. It would make it more difficult to see if the current changes were correct.